test/protractor/helper.js

```diff
@@ -0,0 +1,24 @@
+require('dotenv').load({ path: 'protractor.env', slient: true });
+
+var host = exports.host = process.env.SELENIUM_EPOCHTALK_HOST + ':' + process.env.SELENIUM_EPOCHTALK_PORT;
```

Comment:
can you explain why you used `exports` instead of `module.exports`?

Reply:
cuz https://medium.freecodecamp.org/node-js-module-exports-vs-exports-ec7e254d63ac

Reply:
are there more sources that agree with this?  perhaps someone in the node community

Reply:
Using exports because its more convenient and you don't have to export a wrapper object containing the methods. It just shortens the code and makes it more readable in my opinion. There are people on both sides of the argument for what is more correct, that's just my reasoning.

This:
```
exports.someFunction = function() { return 'something' };
exports.someOtherFunction = function() { return 'something else' };
```

Versus This:
```
var methods = {
  someFunction: function() { return 'something' },
  someOtherFunction: function() { return 'something else' }
};

module.exports = methods;
```